### PR TITLE
add stacky

### PR DESCRIPTION
**Pull request by inoas**

### CODE REVIEW

**Review by lpil — APPROVED** [review]
Thank you

**Review by lpil — COMMENTED** [review]
Oh sorry. You've not committed the config file change. Could you add that please

File: packages/stacky.gleam [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name = "stacky"
+description = "Stacky ☆ BEAM stack trace in Gleam"
```
[the remark this answers is not in the document]
Comment (inoas) [reply packages/stacky.gleam:2]: used the new gleam code that I did not know of, (gleam run) and it seems it is fine now? Sorry for the noise, I just didn't understand that this changed to have runnable gleam code as part of the repo.

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**